samples/demos/irb/Makefile

```diff
@@ -23,11 +23,11 @@ ercc:
 build: ercc
```

Comment:
if when you call build you execute ercc then why do you have it in all?

Reply:
good catch! Let me clean this up.

Reply:
I've updated the Makefile. Note that we call the `all` target implicitly via a `make test` as triggered from the root directory. This means that, we do not run the irb `build` target when we call `make build` from the root, thus, we lazy build irb only when we test it.